ewc-rfc004-individual-wallet-attestation.md

```diff
@@ -0,0 +1,146 @@
+# EWC RFC004: Individual Wallet Attestation - v1.0
```

Comment:
@ntsbs @lalc Shouldn't this be Individual Wallet Instance Attestation? or Are we removing 'Instance' to stay neutral due to the Implementation Act?

Reply:
I am okay with renaming it to Wallet Unit Attestation